index.html

```diff
@@ -191,6 +191,7 @@ <h3>MathML Element Mappings</h3>
 		  <th><abbr title="User Interface Automation">UIA</abbr></th>
 		  <th><abbr title="Accessibility Toolkit">ATK</abbr></th>
 		  <th><abbr title="macOS Accessibility Protocol">AX API</abbr></th>
+                  <th>[[wai-aria-1.1]]</th>
```

Comment:
HTML-AAM keeps wai-aria as a first row, perhaps we should follow this convention here

Reply:
Sure we can do that, but then I need to add a "n/a" or something to all the other elements to avoid the rows to be shifted...

Reply:
Do you want me to do that?

Reply:
you have a point, perhaps it'd be nice to do that as a followup?

Reply:
Yes, we can do it here or as a follow-up. probably the latter is better to keep the delta of this commit small.

Reply:
sounds good with me

Reply:
Done in https://github.com/w3c/mathml-aam/pull/24